### PR TITLE
Remove oneapibeta from census sample.json file

### PR DESCRIPTION
**Pull request by JoeOster**
# Existing Sample Changes
## Description
removed "c intel/label/oneapibeta " from sample.json

Fixes Issue# 
During validation, MandelbrotOMP sample failed and the solution was removing the sting "c intel/label/oneapibeta " from sample.json file

### CODE REVIEW

**Review by mdbtucker — DISMISSED** [review]

File: DirectProgramming/C++/ParallelPatterns/openmp_reduction/CMakeLists.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 set(CMAKE_CXX_COMPILER "icpx")
-# Set default build type to RelWithDebInfo if not specified
+# Set default build type to RelWithDebInfo if not specified.
```
Comment (mdbtucker) [inline DirectProgramming/C++/ParallelPatterns/openmp_reduction/CMakeLists.txt:2]: This change is completely unrelated and should go in a separate PR.

If you really insist on combining these, please update the PR description.

**Review by mdbtucker — APPROVED** [review]

**Comment by JoeOster** [comment]
Census continues to fail, however, I believe it's an env issue with ci. Per validation, I made the required changes to pass. We need @Sanghavi, Vrushabh H or @raoberman  to look at this further to understand the env issue in ci
